src/dotless.Core/Parser/Parsers.cs

```diff
@@ -879,8 +876,15 @@ public Element Element(Parser parser)
 
             PushComments();
             GatherComments(parser); // to collect, combinator must have picked up something which would require memory anyway
-            Node e = ExtendRule(parser) || parser.Tokenizer.Match(@"[.#:]?(\\.|[a-zA-Z0-9_-])+") || parser.Tokenizer.Match('*') || parser.Tokenizer.Match('&') ||
-                Attribute(parser) || parser.Tokenizer.MatchAny(@"\([^)@]+\)") || parser.Tokenizer.Match(@"[\.#](?=@\{)") || VariableCurly(parser);
+            Node e = ExtendRule(parser) 

```

Comment:
The quickest way to get rid of bugs and get closer to how less.js works is to copy the same logic used in the javascript...
https://github.com/less/less.js/blob/master/lib/less/parser/parser.js#L889

The more the project diverges, the harder it is to port updates..